client/actions/agenda.ts

```diff
@@ -248,54 +248,64 @@ const createPlanningFromEvent = (
     event: IEventItem,
     planningDate: Moment = null,
     agendas: Array<string> = []
-) => (
-    (dispatch) => (
-        dispatch(planning.api.save({}, {
-            event_item: event._id,
-            slugline: stringUtils.convertStringFieldForProfileFieldType(
-                'event',
-                'planning',
-                'slugline',
-                'slugline',
-                event.slugline
-            ),
-            planning_date: planningDate || event._sortDate || event.dates.start,
-            internal_note: stringUtils.convertStringFieldForProfileFieldType(
-                'event',
-                'planning',
-                'internal_note',
-                'internal_note',
-                event.internal_note
-            ),
-            name: stringUtils.convertStringFieldForProfileFieldType(
-                'event',
-                'planning',
-                'name',
-                'name',
-                event.name
-            ),
-            place: event.place,
-            subject: event.subject,
-            anpa_category: event.anpa_category,
-            description_text: stringUtils.convertStringFieldForProfileFieldType(
-                'event',
-                'planning',
-                'definition_short',
-                'description_text',
-                event.definition_short
-            ),
-            ednote: stringUtils.convertStringFieldForProfileFieldType(
-                'event',
-                'planning',
-                'ednote',
-                'ednote',
-                event.ednote
-            ),
-            agendas: agendas,
-            language: event.language,
-        }))
-    )
-);
+) => {
+    const newPlanningItem: Partial<IPlanningItem> = {
```

Comment:
I'm wondering if we should extract this out into a util function, and used both here and https://github.com/superdesk/superdesk-planning/blob/develop/client/api/editor/item_events.ts#L83, so we're only defining this conversion once

Reply:
I agree, but it seems it's being done in a different way in `createPlanningFromEvent` and `addPlanningItem`. Is it possible to drop one and use the remaining one in both places?